chain/chain/src/garbage_collection.rs

```diff
@@ -993,6 +993,7 @@ impl<'a> ChainStoreUpdate<'a> {
             | DBCol::FlatStorageStatus
             | DBCol::Misc
             | DBCol::_ReceiptIdToShardId
+            | DBCol::ShardUIdMapping
```

Comment:
what is the reason for not GC the column? do we expect this mapping to be part of archival data, or the mapping may be needed (and cleaned up later) longer than the 5-epoch GC window?

Reply:
Switched to in memory mapping, resolving this comment.

Reply:
I expect the mapping to be on archival node forever and new entries added to this column on each resharding event.

Reply:
What staffik said, the lifetime of the mapping is handled independently of blocks and epoch like it happens in gc. It depends on tracked shards and isn't tied to any fixed number of epochs.